src/services/codefixes/importFixes.ts

```diff
@@ -650,23 +650,21 @@ namespace ts.codefix {
                 //    but we can still offer completions for it.
                 // - `aliased.parent` will be undefined if the module is exporting `globalThis.something`,

```

Comment:
Does this comment still matter?

Reply:
Yes; this is a separate case where something is missing a parent.